grpc-example/grpc-example-java/build.sbt

```diff
@@ -91,5 +94,5 @@ lagomUnmanagedServices in ThisBuild := Map("helloworld.GreeterService" -> s"http
 
 
 def common = Seq(
-  javacOptions in Compile := Seq("-g", "-encoding", "UTF-8", "-parameters", "-Xlint:unchecked", "-Xlint:deprecation", "-parameters", "-Werror")
```

Comment:
I needed to remove this flag. The grpc generators do use deprecated APIs from Akka (ie: ActorMaterializer).